instrumentation/opentelemetry-instrumentation-asyncpg/src/opentelemetry/instrumentation/asyncpg/__init__.py

```diff
@@ -50,6 +52,10 @@
 from opentelemetry.trace import SpanKind
 from opentelemetry.trace.status import Status, StatusCode
 
+from . import package as pkg
+
+_APPLIED = "_opentelemetry_tracer"
```

Comment:
What's this for?

Reply:
I think it got left over from a rebase. Will confirm and remove